extensions/notebook/src/jupyter/common.ts

```diff
@@ -10,6 +10,7 @@ import * as vscode from 'vscode';
 export interface IServerInstance {
 	readonly port: string;
 	readonly uri: vscode.Uri;
+	attachedEditorCount: number;
```

Comment:
should be readonly

Reply:
Thanks. We don't need this anymore after the refactor but noted 😄